scripts/benchmark_execute_primitives.sh

```diff
@@ -3,9 +3,6 @@
 # Specify the base commands
 base_command="nargo execute --package "
 
-# Generate a sequence of arguments for all primitives
-arguments=("keccak256" "keccak256_100_times" "ecdsa_secp256k1" "compute_merkle_root_depth_4" "compute_merkle_root_depth_32" "verify_proof" "storage_proof_depth_8" "rsa")
-
```

Comment:
Does the script work with the arguments / packages removed?

Running it as is in the PR doesn't gather any benchmarking results on my end.

Reply:
Replied above